com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -648,23 +656,37 @@ public void testBundleResourcesDirs() throws IOException, ConfigurationException
         sub_platform2.mkdir();
         createDefaultFiles(contentRoot);
         createFile(contentRoot, "m.txt", "dummy");
-        createFile(sub_platform1.getAbsolutePath(), "s1-1.txt", "dummy");
-        createFile(sub_platform1.getAbsolutePath(), "s1-2.txt", "dummy");
-        createFile(sub_platform2.getAbsolutePath(), "s2-1.txt", "dummy");
-        createFile(sub_platform2.getAbsolutePath(), "s2-2.txt", "dummy");
+        createFile(sub_platform1.getAbsolutePath(), "s1-root1.txt", "dummy");
```

Comment:
Renamed to help with debugging as the name itself now conveys where it is located